### ISSUE TITLE
CutOffY property doesn't cut off spotsLine

### ISSUE THREAD

**Issue by bohdan1krokhmaliuk** [imaNNeoFighT/fl_chart#478]
Area is cut but spotsLines are not, as the cutOffProperty is added to BarAreaData object -> it should cut both area and spotsLines

```
 BarAreaData(
        show: style.shouldShowBarArea,
        colors: style.areaColor(color),
        applyCutOffY: true,
        cutOffY: _convertRealChartValueToChartPoint(0.0, axisIndex),
        spotsLine: BarAreaSpotsLine(
          show: style.shouldShowBars,
          flLineStyle: FlLine(strokeWidth: widget.barWidth, color: color),
        ),
      ),
```

**Comment by imaNNeoFighT** [comment]
I think it should be optional. It depends on your use-case.
BTW I will add an option to cover your usage.
Thanks!

**Comment by imaNNeoFighT** [comment]
Fixed in [0.12.1](https://github.com/imaNNeoFighT/fl_chart/releases/tag/0.12.1). Check it out!.
Thanks for reporting!
